etc/c/donotoptimizeaway/donotoptimizeaway.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
Is there no way we can do this in D?
Currently there is a huge movement to get rid of `etc/c` ...